nerdlandbot/__main__.py

```diff
@@ -60,6 +62,9 @@ async def on_ready():
             "Not starting YouTube scheduler. Please provide a YOUTUBE_TOKEN in your .env file"
         )
 
-
+guilds_data = await get_all_guilds_data()
```

Comment:
This looks great, but i would also add logic to add a guild flag to the dict in the code that creates a new guild.

Reply:
I think I covered this correctly in the new commit